road_to_fit/src/db/queries/admin-query.ts

```diff
@@ -0,0 +1,29 @@
+import { eq } from 'drizzle-orm'
+import { db } from '..'
+import { adminTable, InsertUser, SelectUser } from '../schema/admin.schema'
+
+// Create admin
+export const createAdmin = async (data: InsertUser) => {
+  await db.insert(adminTable).values(data)
+}
+
+// Delete admin
+export const deleteAdmin = async (email: SelectUser['email']) => {
+  await db.delete(adminTable).where(eq(adminTable.email, email))
+}
+
+// Get all admins
+export const getAdmins = async () => {
+  return await db.select().from(adminTable)
```

Comment:
Do we need to send all the information to the client? You're also sending the password, even though it could be encrypted. pick and choose the data

Reply:
@vnaykvkrm Check these as well

Reply:
Updated